pkg/controller/cronjob/utils.go

```diff
@@ -173,8 +173,8 @@ func getRecentUnmetScheduleTimes(sj batchv1beta1.CronJob, now time.Time) ([]time
 func getJobFromTemplate(sj *batchv1beta1.CronJob, scheduledTime time.Time) (*batchv1.Job, error) {
 	// TODO: consider adding the following labels:
 	// nominal-start-time=$RFC_3339_DATE_OF_INTENDED_START -- for user convenience
-	// scheduled-job-name=$SJ_NAME -- for user convenience
 	labels := copyLabels(&sj.Spec.JobTemplate)
+	labels["cronjob-name"] = sj.ObjectMeta.Name
```

Comment:
In addition to that, I would suggest we at least introduce a constant instead of having a hardcoded key 'cornjob-name'.  

`kubernetes-cornjob-name` or `k8s-cornjob-name` both look good to me.

Reply:
Good point. I've done that.